src/psyclone/tests/lfric_build.py

```diff
@@ -69,12 +69,12 @@ class LFRicBuild(Compile):
                               "stencil_dofmap_mod",
                               "function_space_mod",
                               "field_mod",
-                              "abstract_quadrature_mod",
-                              "quadrature_rule_mod",
+                              "quadrature_mod",
                               "quadrature_xyz_mod",
                               "quadrature_xyoz_mod",
                               "quadrature_xoyoz_mod",
-                              "quadrature_mod",
+                              "quadrature_edge_mod",
+                              "quadrature_face_mod",
                               "operator_mod",
                               "flux_direction_mod"]
 
```

Comment:
I generally aim to do imports as locally as possible in order to avoid cyclic dependencies and help developers see where a symbol has come from. This pylint warning has only appeared in the latest release of pylint and we don't yet have a strategy for dealing with it.